actionview/lib/action_view/layouts.rb

```diff
@@ -405,11 +405,11 @@ def _normalize_layout(value)
     #
     # ==== Parameters
     # * <tt>formats</tt> - The formats accepted to this layout
-    # * <tt>require_layout</tt> - If set to true and layout is not found,
-    #   an +ArgumentError+ exception is raised (defaults to false)
+    # * <tt>require_layout</tt> - If set to +true+ and layout is not found,
+    #   an +ArgumentError+ exception is raised (defaults to +false+)
```

Comment:
Oh yeah, I noticed that we don't have a test for `_default_layout` raising an error when `require_layout` is set to `true` 😄